apps/ledger-live-desktop/src/renderer/modals/Platform/Exchange/CompleteExchange/Body.tsx

```diff
@@ -198,10 +195,6 @@ const Body = ({ data, onClose }: { data: Data; onClose?: () => void | undefined
 
   const onBroadcastSuccess = useCallback(
     (operation: Operation) => {
-      if (getEnv("DISABLE_TRANSACTION_BROADCAST")) {
-        return onCancel(new DisabledTransactionBroadcastError());
-      }
-
```

Comment:
I think we'd still need a mecanism to cancel swaps when DISABLE_TRANSACTION_BROADCAST is on. 
The full error might not have been the solution to tests with it. 

Could we just just return the info somehow to the live app to make sure we tell our partner to not await for the swap. (Passing the env to the live-app at start up or inject it inside the operation ...)  Especially since I know some of you do not override your env and dev with prod quote requests :)